.github/docker/quick-lint-js-github-website-builder/README.md

```diff
@@ -0,0 +1,13 @@
+# quick-lint-js-website-builder Docker image
+
+## Updating the Docker image
+
+Pick a version number for the new Docker image (e.g. `v3`), then run the
+following commands:
+
+    $ docker build --tag ghcr.io/quick-lint/quick-lint-js-website-builder:VERSION_NUMBER_HERE .github/docker/quick-lint-js-website-builder/
```

Comment:
The name of the image is incorrect (missing `github-`). I'll fix it when I merge.